deploy/charts/library/templates/_cluster-role.tpl

```diff
@@ -148,4 +148,15 @@ rules:
   - apiGroups: [""]
     resources: ["persistentvolumeclaims"]
     verbs: ["get", "update", "delete", "list"]
+---
+# Aspects of ceph osd purge job that require access to the cluster namespace
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: rook-ceph-default
+  namespace: {{ .Release.Namespace }} # namespace:cluster
+rules:
```

Comment:
If any roles are added to the service account, then it's not the default. Anything that requires non-default privileges should have a separate service account.

Reply:
that comment was added by mistake